19 show my accounts

ZacTredger opened the pull request:
After we discussed refactoring the way the system handles money, I've decided that's best-placed in a different issue/PR (which I'll whack on Trello). So this continues to use doubles.

Thanks gang,
Zac

juliano60 commented:
Ah missed this PR. The changes look fine, though a bit out of date with "main". Happy for you to merge after catching up with main and resolving any conflict.

Thanks and Regards,
Jules

ZacTredger committed:
Accounts have balances not openingBalances

ZacTredger committed:
Display account statements on separate lines

Previously, statements from multiple accounts were concatenated with no
delimiter. This fix updates the relevant integration test and adds a
test to confirm that SHOWMYACCOUNTS doesn't work if the user isn't
logged in.

michowski reviewed (commented):
Are we planning to merge this one?

ZacTredger commented:
> Are we planning to merge this one?
I was just preparing myself to plunge into rebase hell, but pleased to discover there are no conflicts!
